_garden/love-letter-to-early-internet.md

```
@@ -71,158 +217,113 @@ And of course, an updated dogblog button from 20**25**:
-The internet from *before* the late 90s was a much more static place because the technology to create and share animations wasn't as widespread. Graphics programs were rarer because PCs were less-powerful, and bandwidth was much lower and more-expensive. Even if you *did* make an awesome animation, your visitors might not be able to pull it down and see it - or they might not *appreciate* having to! So once people started being **able** to do it, they were off to the races doing it everywhere! Buttons, decorations, "look at this cool pic I found," and yes, even *banner ads* blinked and vied for your attention everywhere... and we (except for the banner ads), generally loved it. The web was coming *alive* and it felt *cool*!
-
-Feast your eyes on this bad boy:
+From there, the thirst for traffic birthed a sibling idea: reciprocal "Affiliate Links." Long before "affiliates" tooks its current revenue-sharing meaning, "Affiliates" were just other sites that you agreed to swap links with, and you'd each display them somewhere prominent. That's what your [88x31](#88x31--468x60) buttons were for!
 
 {% polaroid
-	early-web/90s-site-with-animation-everywhere.gif
-	title="A 90s website with animated GIFs all over the place"
-	link="https://geocities.restorativland.org/SiliconValley/Network/1666/"
-	archive="https://web.archive.org/web/20200305030854/https://geocities.restorativland.org/SiliconValley/Network/1666/"
+	./early-web/affiliates-ghpd-2003.jpg
+	title="Affiliates on Gengar and Haunter's Pokemon Dungeon - 2003"
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)